sirepo/template/raydata.py

```diff
@@ -121,10 +99,14 @@ def stateless_compute_scans(data):
             raise sirepo.util.UserAlert(
                 f'More than {_MAX_NUM_SCANS} scans found. Please reduce your query.',
             )
-        s.append(_scan_info(v[0], metadata=v[1].metadata))
+        s.append(_scan_info(v[0], data.selectedColumns, metadata=v[1].metadata))
     return _scan_info_result(s)
 
 
+def stateless_compute_get_scan_fields(data):
```

Comment:
I think the `get` can be removed from the name (this is more personal preference so feel free to push back but it is redundant). Also, sort the method alphabetically.